unit-tests/run-unit-tests.py

```diff
@@ -132,11 +132,14 @@ def subprocess_run(cmd, stdout = None):
         if stdout  and  stdout != subprocess.PIPE:
             handle = open( stdout, "w" )
             stdout = handle
+        start_time = time.time()
         rv = subprocess.run( cmd,
                              stdout = stdout,
                              stderr = subprocess.STDOUT,
                              universal_newlines = True,
+                             timeout=200,
                              check = True)
+        run_time = time.time() - start_time

```

Comment:
What happens in case of a timeout?
Is the process still running? Is it killed?
Please document behavior for the function

Reply:
is the explanation in the doc enough?